tracer/src/Datadog.Trace.Tools.dd_dotnet/Checks/Resources.cs

```diff
@@ -46,6 +46,7 @@ internal static class Resources
         public const string DdAgentChecks = "---- DATADOG AGENT CHECKS -----";
 
         public const string ContinuousProfilerEnabled = "DD_PROFILING_ENABLED is set.";
+        public const string ContinuousProfilerSsiEnabled = "DD_INJECTION_ENABLED is set. Profiler is enabled through SSI.";
```

Comment:
A follow-up from the previous comment: should be Profiler is **deployed** through SSI instead of **enabled**